src/components/views/messages/MFileBody.js

```diff
@@ -21,7 +21,42 @@ import filesize from 'filesize';
 import MatrixClientPeg from '../../../MatrixClientPeg';
 import sdk from '../../../index';
 import {decryptFile} from '../../../utils/DecryptFile';
-
+import Tinter from '../../../Tinter';
+
+// A cached tinted copy of "img/download.svg"
+var tintedDownloadImageURL;
+// Track a list of mounted MFileBody instances so that we can update
+// the "img/download.svg" when the tint changes.
+var nextMountId = 0;
+const mounts = {};
+
+/**
+ * Updates the tinted copy of "img/download.svg" when the tint changes.
+ */
```

Comment:
Nope, I'm using it because I couldn't work out how one is supposed to use fetch to get an XML document.